src/utils/custom_yaml.py

```diff
@@ -0,0 +1,10 @@
+import json
+import re
+import yaml
+
+def loads (content, **args):
+    contents=''
+    for line in content.split('\n'):
+        if not re.match (r'\s*//.*', line):
+            contents += line + "\n"
```

Comment:
I believe this is not needed. Yaml supports comments by default (in the specification and in the library)